airbyte-webapp/src/types/react-table.d.ts

```diff
@@ -7,7 +7,9 @@ import "@tanstack/react-table";
  */
 declare module "@tanstack/table-core" {
   interface ColumnMeta {
-    thClassName?: string;
+    collapse?: boolean;
+    headerHighlighted?: boolean;
```

Comment:
I'm not sure I understand what is `headerHighlighted`. 
Can we style instead?

Reply:
Agree, this should probably be styled using a classname.
I have a separate Issue that I created to clean up the styling on the tables (header alignment, header highlighting, etc.) https://github.com/airbytehq/airbyte/issues/21941